fastify/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const { extension } = require('../extension');
+const setupRoutes = require("./routes");
+const { applicationProxyRoutes } = require("./api_routes");
+const logger = require('../logger');
+const { isAuthorized, getApplicationConfig } = require('../middleware/session_middleware');
+const routerHandlers = require('../handlers');
+const { getApplicationClient, getPlatformClient } = require('../utils');
+
+function setupFdk(data, syncInitialization) {
+    if (data.debug) {
+        logger.transports[0].level = 'debug';
+    }
+    const promiseInit = extension.initialize(data)
+        .catch(err=>{
+            logger.error(err);
+            throw err;
+        });
+
+    const configInstance =  {
+        fdkHandler: setupRoutes,
```

Comment:
Are we not exposing apiRoutes here ?

Reply:
No we are not exposing it as it was implementing session cookies auth for, which we need to remove